src/variables/variable_common.jl

```diff
@@ -52,9 +52,33 @@ function add_variable!(
             ind => _variable(m, name, ind, lb, ub, bin, int)
             for ind in indices(m; t=vcat(history_time_slice(m), time_slice(m)))
         )
+    if use_representative_days(model=m.ext[:instance]) == :use_representative_days_w_seasonal_storage
+        map_variable!(m, m.ext[:variables][name], indices)
+    end
     ((bin != nothing) || (int != nothing)) && push!(m.ext[:integer_variables], name)
 end
 
+"""
+    map_variable!(v::Dict{VariableRef}, indices_all::Function)
```

Comment:
maybe the name could be `replace_non_representative_periods!(...)` or `map_periods!(...)`? I don't know, `map_variable!` sounds a bit odd. Maybe `remap_variable!` ???